src/components/molecules/MyComponent/MyComponent.tsx

```diff
@@ -1,9 +1,17 @@
 import { CustomDatePicker } from "@/components/atoms/DatePicker/DatePicker";
 
 export function MyComponent() {
+  const endDateLimit = new Date();
+  endDateLimit.setDate(endDateLimit.getDate() + 30);
   return (
     <div className="w-full flex justify-center">
-      <CustomDatePicker isEndDate />
+      <CustomDatePicker isEndDate 
+                        startDatePlaceHolder="Start Date"
+                        endDatePlaceHolder="End Date"
+                        calendarStartFromMondayDay={false}
```

Comment:
This can be the same as a `true` flag if we won't provide props it'll take `false`.

Reply:
Please read esLint suggestions for examples.
```JavaScript
 <CustomDatePicker isEndDate 
                        startDatePlaceHolder="Start Date"
                        endDatePlaceHolder="End Date"
                        //calendarStartFromMondayDay={false} => It'll be false
                        endDateLimit={endDateLimit}
                        markSundayAsRed // It'll be true
                          />

Reply:
got it; thanks;